azure/template.json

```
@@ -144,6 +144,25 @@
       "metadata": {
         "description": "API Key for Google Geocode"
       }
+    },
+    "settingsCycleEndingSoon": {
+      "type": "string",
```

```suggestion
      "type": "bool",
      "defaultValue": false
```
could do the same (`type: bool` and `defaultValue: false`) for other parameters too?

---

Done! 👍